### PR TITLE
Correction of urls for edisp and rad_max IRFs

### PR DESCRIPTION
**Pull request by GernotMaier**
Very minor corrections of urls to GADF documentation.

### CODE REVIEW

File: pyirf/io/gadf.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ def create_rad_max_hdu(
     """
     Create a fits binary table HDU in GADF format for the directional cut.
     See the specification at
-    https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/aeff/index.html
+    https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/index.html?highlight=RAD_MAX#rad-max
```
Comment (HealthyPear) [inline pyirf/io/gadf.py:271]: apparently you copy-paste the url while you were searching for "RAD_MAX"

it should be just

https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/index.html#rad-max

**Review by HealthyPear — CHANGES_REQUESTED** [review]
thanks for the catch!

**Comment by HealthyPear** [comment]
Ah I forgot: due to the download of the test data we do not support anymore PRs from forked branches.
I think you should close this PR and reopen it from a branch

**Comment by GernotMaier** [comment]
> Ah I forgot: due to the download of the test data we do not support anymore PRs from forked branches.
> I think you should close this PR and reopen it from a branch

I tried, but I can't push from a branch.

**Comment by HealthyPear** [comment]
> I tried, but I can't push from a branch.

I think it's because you had no write permission (I saw you are not in the cta-dev group for example).
I added you as a single user just now - try again

**Comment by maxnoe** [comment]
@GernotMaier I added you to the https://github.com/orgs/cta-observatory/teams/cta-devs/members group, you should be able to push without a fork now.

**Comment by GernotMaier** [comment]
Great, thanks. I will delete this pull request and open another one.

**Pull request closed without merging**